core/templates/components/forms/custom-forms-directives/thumbnail-display.component.html

```diff
@@ -3,3 +3,10 @@
      [style.height]="height"
      [style.width]="width"
      [style.background]="background">
+
+<style>
+  img.img-thumbnail.protractor-test-custom-photo {
+    height: 120px !important;
+    width: 120px !important;
```

Comment:
sure about the use of `!important`? If your styles are getting overwritten. Then make the CSS selectors more specific.

Reply:
Don't use !important... This will help you: https://css-tricks.com/specifics-on-css-specificity/